src/node/constants.ts

```diff
@@ -23,7 +23,24 @@ export const version = pkg.version || "development"
 export const commit = pkg.commit || "development"
 export const rootPath = path.resolve(__dirname, "../..")
 export const vsRootPath = path.join(rootPath, "vendor/modules/code-oss-dev")
+export const codeVersion = require(path.join(vsRootPath, "package.json")).version
 export const tmpdir = path.join(os.tmpdir(), "code-server")
 export const isDevMode = commit === "development"
 export const httpProxyUri =
   process.env.HTTPS_PROXY || process.env.https_proxy || process.env.HTTP_PROXY || process.env.http_proxy
+
+// getVersionString returns a human-readable version string suitable
+// for outputting to the console.
+export function getVersionString(): string {
```

Comment:
exporting these functions means we can test in constants.test.ts, and then the entry.ts code just needs to call these (and thus does not need to be tested)

this is part 1, the second part will change this to return the code-server version info and change the entry.ts code to call these functions. I split these up to streamline review.

Reply:
nit: if you use the TSDoc block syntax, IDEs will show this information on hover.

<img width="520" alt="image" src="https://user-images.githubusercontent.com/3806031/156057740-89b56cdb-c293-494e-b114-dd5ba941ffa4.png">


```typescript
/**
 * getVersionJsonString returns a machine-readable version string
 * suitable for outputting to the console.
 */
```

Reply:
Great tip, will update!